sirepo/package_data/static/json/controls-schema.json

```diff
@@ -54,6 +54,21 @@
             ["vertical", "Vertical"]
         ]
     },
+    "frameIdFields": {
+        "instrumentAnimation": [
+            "x",
+            "y1",
+            "y2",
+            "y3",
+            "histogramBins",
+            "plotRangeType",
+            "horizontalSize",
+            "horizontalOffset",
+            "verticalSize",
+            "verticalOffset",
+            "refreshId"
```

Comment:
It looks like refreshId is important for making the heatmap api calls. Without it the heatmaps don't update at all during the optimization

Reply:
Ah thanks for looking into it! Thinking about it more it makes sense. It is a bit of randomness added to the url for the frame which causes a full request to the server because the cached request in the browser has a different url.

Reply:
should we use something other than
```javascript
appState.models[m].refreshId = Math.random();
```
?

like a hash or a random integer rather than the float generated by Math.random()?
Is there any potential issue with the api call parser seeing the '.' in the float that we use as a refreshId?

Reply:
The '.' won't break anything, but really it could be removed and set frameCount to a simulation run-count or similar to make the query unique. That would require additional changes so the playback buttons would remain hidden on these "single-frame" reports. For now Math.random() is good enough.

Reply:
👍